README.md

```diff
@@ -71,6 +71,189 @@ const result = validate(value, validations)
 | null          |                   |                   |  
 | undefined     |                   |                   |  
 
+
+
+#### Contains
+
+`contains` -  The contains validator is useful for validating allowance or restriction in certain values.
+It checks that the given value exists in the target given by the **allowed** or  **notAllowed** option.
+
+You can specify the validator as a list, string or as an object (in which case the keys of the object are used).
+
+**allowed option examples:**
+
+```javascript
+var list = ["small", "medium", "large"]
+const value = 'small'
+var options = { allowed: list }
+const validations = { contains: options }
+const result = validate(value, validations) 
+/* {
+    value: 'small',
+    errors: []
+} */
+
+var list = ["small", "medium", "large"]
+const value = 'xlarge'
```

Comment:
to favor the didactics of the example, I would make this line simpler and straightforward:
```javascript
const value = 'xlarge'
const validations = { contains: { allowed: ["small", "medium", "large"] } }
```

also, use `const` instead of `var`

Reply:
I also modified the test files

Reply:
It seems the change wasn't made

Reply:
My mistake, I just saw the const intead var here. I'll change this one